core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1799,18 +1816,23 @@ write_files:
                 command:
                 - /hyperkube
                 - proxy
-                - --kubeconfig=/etc/kubernetes/kubeconfig/kube-proxy.yaml
-                - --cluster-cidr={{.PodCIDR}}
+                - --config=/etc/kubernetes/kube-proxy/kube-proxy-config.yaml
```

Comment:
Very nit but I'd prefer `kube-proxy/config.yaml` for less repeating.
If you could agree, I'd appreciate it if you could address it when you work on the config afterwards.. probably for the ipvs support? 😉